pkg/util/capabilities/file.go

```diff
@@ -11,6 +11,8 @@ import (
 	"io/ioutil"
 	"os"
 	"syscall"
+
+	"github.com/sylabs/singularity/internal/pkg/runtime/engines/config"
```

Comment:
As I mentioned in the other comment, we shouldn't be importing `internal/pkg/runtime/...` from a package in `pkg/...`. This would (likely) break any external thing importing this package